gcc/rust/parse/rust-parse-impl.h

```diff
@@ -507,7 +507,12 @@ AST::Attribute
 Parser<ManagedTokenSource>::parse_inner_attribute ()
 {
   if (lexer.peek_token ()->get_id () == INNER_DOC_COMMENT)
-    return parse_doc_comment ();
+    {
+      auto values = parse_doc_comment ();
+      return AST::Attribute (std::move (std::get<0> (values)),
+			     std::move (std::get<1> (values)),
+			     std::get<2> (values), true);
```

Comment:
I think this would be a bit clearer if you destructured the tuple
```suggestion
      // probably missing some `&&` or std::move somewhere
      auto path = std::move(std::get<0>(values));
      auto input = std::move(std::get<1>(values));
      auto loc = std::get<2>(values);
      return AST::Attribute (path, input, loc, true);
```

Reply:
Fixed.